tcf_core/settings/base.py

```diff
@@ -7,7 +7,10 @@
 import environ
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+BASE_DIR = os.path.dirname(
+    os.path.dirname(
+        os.path.dirname(
+            os.path.abspath(__file__))))
```

Comment:
Seems like there might have been some inadvertent reformatting issues? This happened to me too when I ran precommit but I'm not too sure what the solution is. From what I've been told, I think we're supposed to discard such changes if possible (which is pretty easy to do with Github Desktop if you don't already use it).